app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -409,6 +436,52 @@ export default {
         },
       });
     },
+    isEditorMouseFocusedOnAnImage() {
+      const selectedNode = document.querySelector(
+        'img.ProseMirror-selectednode'
+      );
+      if (selectedNode) {
+        this.selectedNode = selectedNode;
+        this.isImageSelected = !!selectedNode;
+        // Get the position of the selected node
+        this.setToolbarPosition();
+      } else {
+        this.isImageSelected = false;
+      }
+    },
+    setToolbarPosition() {
+      const editorRoot = document.querySelector('.editor-root');
```

Comment:
Can we set this value at the start on mounted? We won't have to do the query all the time then. Also, if possible can we use `$ref` instead?